rusoto/credential/src/lib.rs

```diff
@@ -33,73 +19,38 @@ pub mod claims;
 mod container;
 mod environment;
 mod instance_metadata;
-mod object_safe;
 mod profile;
 mod request;
 mod secrets;
 mod static_provider;
+#[cfg(test)]
 pub(crate) mod test_utils;
 mod variable;
 
+use async_trait::async_trait;
 use std::collections::BTreeMap;
 use std::env::{var as env_var, VarError};
 use std::error::Error;
 use std::fmt;
 use std::io::Error as IoError;
-use std::ops::Deref;
-use std::rc::Rc;
 use std::string::FromUtf8Error;
-use std::sync::{Arc, Mutex};
+use std::sync::Arc;
 use std::time::Duration;
 
 use chrono::{DateTime, Duration as ChronoDuration, ParseError, Utc};
-use futures::future::{err, Either, Shared, SharedItem};
-use futures::{Async, Future, Poll};
 use hyper::Error as HyperError;
-
-/// Representation of anonymity
-pub trait Anonymous {
```

Comment:
Is this replaced by some code? Looks just removed.

Reply:
I believe this is intended to be removed, yes.

Reply:
That code was added in https://github.com/rusoto/rusoto/pull/1566. Do anonymous credentials still work?